.envrc

```diff
@@ -1 +1 @@
-use flake
+use flake . --accept-flake-config
```

Comment:
As mentioned, I believe that `--accept-flake-config` can be removed now that you removed the `nixConfig` stanza from `flake.nix`.
```suggestion
use flake
```